src/utils/DataConfig.py

```diff
@@ -99,16 +194,19 @@ def __init__(self, dataset_path: str):
             5: "sandwich time",
         }
 
-    def load_dataset(self) -> "list[Recording]":
+    def _load_dataset(self) -> "list[Recording]":
         return load_opportunity_dataset(self.dataset_path)
 
+    def _getDataConfigIdentifier(self):
```

Comment:
same method as in parent class, why do we need to declare it again?

Reply:
Historisch gewachsen